posthog/models/subscription.py

```diff
@@ -123,6 +121,33 @@ def url(self):
             return absolute_uri(f"/dashboard/{self.dashboard.id}/subscriptions/{self.id}")
         return None
 
+    @property
+    def summary(self):
+        human_frequency_map = {
+            "daily": "day",
+            "weekly": "week",
+            "monthly": "month",
+            "yearly": "year",
+        }
+
+        human_bysetpos_map = {
+            1: "first",
+            2: "second",
+            3: "third",
+            4: "fourth",
+            -1: "last",
+        }
+
+        frequency = human_frequency_map[self.frequency]
```

Comment:
Do we know it is impossible for `self.frequency` to be an invalid value?

I wonder if we should catch and en-nicen the error if there's an invalid `self.frequency`

Reply:
It shouldn't be but rather than crash the whole app if it is I will change it to a `.get()`  👍